### PR TITLE
Add man page

### PR DESCRIPTION
**Pull request by evertiro**
When I added the PPA for Gcolor3, I realized you don't currently have a man page. This is technically a requirement for Debian-based distros.

### CODE REVIEW

File: man/gcolor3.1 [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.TH GCOLOR3 "1" "August 26, 2018"
+.
+.SH NAME
+gcolor3 \- color selection dialog
```
Comment (Hjdskes) [inline man/gcolor3.1]: Please make this "a color selection dialog"

File: man/gcolor3.1 [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+.PP
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+.
+.SH BUGS
+.SS "Reporting Bugs"
```
Comment (Hjdskes) [inline man/gcolor3.1]: Please remove this line.

File: man/gcolor3.1 [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+.TP
+.BR \-h ", " \-\^\-help
+Show help options
+.TP
+.BR \-\^\-help\-all
+Show all help options
+.TP
+.BR \-\^\-help\-gtk
+Show GTK+ options
+.SS "GTK+ Options"
```
Comment (Hjdskes) [inline man/gcolor3.1]: I viewed some man pages of other GNOME applications, and none included these. Can you please remove them?

File: man/gcolor3.1 [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+GTK+ debugging flags to unset
+.SS "Application Options"
+.TP
+.BR \-v ", " \-\^\-version
+Show the application's version
+.TP
+.BI "\-\^\-display=" DISPLAY
+X display to use
+.
+.SH COPYRIGHT
```
Comment (Hjdskes) [inline man/gcolor3.1]: Is it common to include the full license in man pages?

File: meson.build [hunk 5]
```diff
@@ -43,6 +43,8 @@ config_h.set('UNUSED', '__attribute__ ((unused))')
 configure_file(output: 'config.h',
   configuration: config_h)
 
+install_man('man/gcolor3.1')
```
Comment (Hjdskes) [inline meson.build]: Can you please move the man page to the data folder? No need for a separate folder.

Comment (Hjdskes) [reply meson.build]: When you do, please move the `install_man` line to `data/meson.build` as well.

File: man/gcolor3.1 (hunk 3 above)
Comment (evertiro) [reply man/gcolor3.1]: Only included them because --help did, but no problem!

File: man/gcolor3.1 (hunk 4 above)
Comment (evertiro) [reply man/gcolor3.1]: That's not the full license, it's the recommended subset of the GPL to include in a single file. The full license is MUCH longer.

Full license: https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
Reference for applying: https://www.gnu.org/licenses/gpl-2.0.en.html#SEC4

**Comment by Hjdskes** [comment]
Thanks :) I checked a few popular man pages (i.e., vim, bash, man itself) and none include (a subset of) the license. Can you switch that to just an AUTHORS section instead? You can make it "Jente Hidskes and contributors".

**Comment by evertiro** [comment]
Done

**Comment by evertiro** [comment]
You're helping me, I'll help you... with that tiny addition, it's valid for Debian/Ubuntu!

**Pull request merged**